utils/functions.sh

```diff
@@ -182,13 +182,15 @@ install_conda_shell_scripts() {
         echo "#!/bin/sh" > "$prefix/$BIN_DIR/activate"
         echo "_CONDA_ROOT=\"\$(cygpath '$WIN_PREFIX')\"" >> "$prefix/$BIN_DIR/activate"
         cat "$src_dir/conda/shell/bin/activate" >> "$prefix/$BIN_DIR/activate"
-        chmod +x "$prefix/$BIN_DIR/activate"
+        # chmod +x "$prefix/$BIN_DIR/activate"
+        # we no longer set activate as executable, to ensure that it is being sourced
```

Comment:
The same changes should also be applied to the non-Windows case below

Reply:
Done!